### DISCUSSION TITLE
Saving each iteration as a separate output

### DISCUSSION THREAD

**Discussion by Siddhant-Bharadwaj** [rkhamilton/vqgan-clip-generator#72] [Q&A]
Is there any way I can save each image iteration as a separate image along the process rather than just the final output? Thanks.

**Comment by rkhamilton** [comment]
You didn’t say whether you are making videos or single images, but you will find the [table of common parameters](https://github.com/rkhamilton/vqgan-clip-generator#parameters-common-to-vqgan_clipgenerate) useful. If you set save_every to 1, you will get a saved image every iteration.

**Reply by Siddhant-Bharadwaj** [reply]
I intended to use single text prompt with 1000 iterations to get output image saves at every 10th image but I get only one single output image instead of seperate 100 .jpeg images. Is there any way I could get every 10th image saves as a seperate image files rather than a single output image. Thanks.
